network-api/networkapi/templates/fragments/buyersguide/pni_category_nav.html

```diff
@@ -1,6 +1,6 @@
 {% load bg_nav_tags localization i18n static wagtailroutablepage_tags wagtailmetadata_tags %}
 <nav id="multipage-nav" class="pni-category-nav text-center d-none d-md-block tw-no-scrollbar" title="{% trans "site navigation" context "Tooltip on menu items" %}">
-    <div class="container tw-py-2" id="product-review">
+    <div class="container tw-py-2" id="buyersguide-category-container">
       <div class="row">
         <div class="col">
           <div id="pni-category-wrapper" class="tw-flex tw-items-center tw-w-max tw-min-w-full">
```

Comment:
These ids (`buyersguide-category-container` and `pni-category-wrapper`) are confusingly similar. Do we need a wrapper and a container? Could we use one in place of the other or is there a crucial difference I am missing?

Reply:
We need both of these for `pni-category-dropdown.js` and they are both essential for calculating what should be added to the dropdown.

Reply:
Are there any more expressive names we could use?